src/Microsoft.FileFormats/PDB/PDBFile.cs

```diff
@@ -14,19 +14,22 @@ public class PDBFile
         private readonly Lazy<PDBFileHeader> _header;
         private readonly Lazy<Reader[]> _streams;
         private readonly Lazy<PDBNameStream> _nameStream;
+        private readonly Lazy<DbiStream> _dbiStream;
 
         public PDBFile(IAddressSpace dataSource)
         {
             _reader = new Reader(dataSource);
             _header = new Lazy<PDBFileHeader>(() => _reader.Read<PDBFileHeader>(0));
             _streams = new Lazy<Reader[]>(ReadDirectory);
             _nameStream = new Lazy<PDBNameStream>(() => new PDBNameStream(Streams[1]));
+            _dbiStream = new Lazy<DbiStream>(() => new DbiStream(Streams[3]));
         }
 
         public PDBFileHeader Header { get { return _header.Value; } }
         public IList<Reader> Streams { get { return _streams.Value; } }
         public PDBNameStream NameStream { get { return _nameStream.Value; } }
-        public uint Age { get { return NameStream.Header.Age; } }
+        public DbiStream DbiStream { get { return _dbiStream.Value; } }
+        public uint Age { get { return DbiStream.Header.Age; } }
```

Comment:
Given that there are two Ages in the file, maybe change the name to DbiAge so its clear which one is being refered to?

Reply:
I added the DbiAge property like you suggested.